Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -255,16 +255,7 @@ private void OnElectrifiedInteractUsing(EntityUid uid, ElectrifiedComponent elec
             }
         }
 
-        /// <param name="uid">Entity being electrocuted.</param>
-        /// <param name="sourceUid">Source entity of the electrocution.</param>
-        /// <param name="shockDamage">How much shock damage the entity takes.</param>
-        /// <param name="time">How long the entity will be stunned.</param>
-        /// <param name="refresh">Should <paramref>time</paramref> be refreshed (instead of accumilated) if the entity is already electrocuted?</param>
-        /// <param name="siemensCoefficient">How insulated the entity is from the shock. 0 means completely insulated, and 1 means no insulation.</param>
-        /// <param name="statusEffects">Status effects to apply to the entity.</param>
-        /// <param name="ignoreInsulation">Should the electrocution bypass the Insulated component?</param>
-        /// <returns>Whether the entity <see cref="uid"/> was stunned by the shock.</returns>
-        public bool TryDoElectrocution(
```

Comment:
no need to remove all the comments here. just put them on the shared one and do <inheritdoc>

Reply:
do inheritdoc???

Reply:
gonna assume thats what you meant since cut off